Tests/LibSQL/TestSqlDatabase.cpp

```diff
@@ -176,13 +176,3 @@ TEST_CASE(insert_100_into_table)
 {
     insert_and_verify(100);
 }
-
-TEST_CASE(insert_1000_into_table)
```

Comment:
These are good candidates for `BENCHMARK_CASE` from LibTest/TestCase.h. that way they'll run locally unless you specify the "no benchmarks" flag/environment variable.